### ISSUE TITLE
Yolov5 change silu to relu accuracy problem after exporting to onnx.

### ISSUE THREAD

**Issue by maro-jeon** [NVIDIA-AI-IOT/yolov5_gpu_optimization#11]
Hi, 
I saw your study about changing activation function from silu to relu.
The score shows that ReLU mAP is 55.7 (swish pretrained).
so I just changed swish activation to relu activation without any fine-tuning process.

from
![image](https://github.com/NVIDIA-AI-IOT/yolov5_gpu_optimization/assets/135101992/bf084916-49af-422e-a4a6-5f253fcafb31)
to
![image](https://github.com/NVIDIA-AI-IOT/yolov5_gpu_optimization/assets/135101992/9e324c22-ffec-4681-af42-c785f6e7a9fe)

Then, the result of accuracy was zero.

Can you describe more about how to achieve the score 55.7 (swish pretrained) for reproducing the score.

Thanks,
Maro JEON

**Comment by Tyler-D** [comment]
You have to retrain the model.

**Comment by maro-jeon** [comment]
Thanks, you just fine tune ? or train many epochs ?

**Comment by Tyler-D** [comment]
Train for 300 epochs.
